commit dd05fdd35f680dfff606731a664859ae45762bb1
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Mon Mar 26 13:03:03 2012 +0300

    Dont bother building internal versions of db utilities we dont need
    
    - rpmdb_dump, load, recover, verify, stat etc are useful at times,
      but these are not. This also fixes build with internal db for
      more recent versions of Berkeley DB.

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 8c3b970..2245c19 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -82,29 +82,6 @@ CLEANFILES += $(libdb_la)
 
 rpmlibexec_PROGRAMS =
 
-rpmlibexec_PROGRAMS += rpmdb_archive
-rpmdb_archive_SOURCES =
-rpmdb_archive_LDADD = \
-	$(top_builddir)/db3/db_archive.o \
-	$(top_builddir)/db3/util_sig.o \
-	librpm.la
-
-rpmlibexec_PROGRAMS += rpmdb_checkpoint
-rpmdb_checkpoint_SOURCES =
-rpmdb_checkpoint_LDADD = \
-	$(top_builddir)/db3/db_checkpoint.o \
-	$(top_builddir)/db3/util_log.o \
-	$(top_builddir)/db3/util_sig.o \
-	librpm.la
-
-rpmlibexec_PROGRAMS += rpmdb_deadlock
-rpmdb_deadlock_SOURCES =
-rpmdb_deadlock_LDADD = \
-	$(top_builddir)/db3/db_deadlock.o \
-	$(top_builddir)/db3/util_log.o \
-	$(top_builddir)/db3/util_sig.o \
-	librpm.la
-
 rpmlibexec_PROGRAMS += rpmdb_dump
 rpmdb_dump_SOURCES =
 rpmdb_dump_LDADD = \
@@ -121,22 +98,6 @@ rpmdb_load_LDADD = \
 	$(top_builddir)/db3/util_sig.o \
 	librpm.la
 
-rpmlibexec_PROGRAMS += rpmdb_printlog
-rpmdb_printlog_SOURCES =
-rpmdb_printlog_LDADD = \
-	$(top_builddir)/db3/db_printlog.o \
-	$(top_builddir)/db3/btree_autop.o \
-	$(top_builddir)/db3/crdel_autop.o \
-	$(top_builddir)/db3/db_autop.o \
-	$(top_builddir)/db3/dbreg_autop.o \
-	$(top_builddir)/db3/fileops_autop.o \
-	$(top_builddir)/db3/hash_autop.o \
-	$(top_builddir)/db3/qam_autop.o \
-	$(top_builddir)/db3/rep_autop.o \
-	$(top_builddir)/db3/txn_autop.o \
-	$(top_builddir)/db3/util_sig.o \
-	librpm.la
-
 rpmlibexec_PROGRAMS += rpmdb_recover
 rpmdb_recover_SOURCES =
 rpmdb_recover_LDADD = \
